web/Views/Shared/_Layout.cshtml

```diff
@@ -40,13 +40,13 @@
                     <ul class="navbar-nav ms-auto">
                         @if (User.Identity?.IsAuthenticated == true)
                         {
-                            <li class="nav-item" style="margin-left: auto;">
+                            <li class="nav-item">
                                 <a class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
                             </li>
-                            <li class="nav-item" style="margin-left: auto;">
+                            <li class="nav-item">
                                 @using (Html.BeginForm("Logout", "Login", FormMethod.Post))
                                 {
-                                    <button type="submit" class="btn btn-link nav-link">Log out</button>
+                                    <button type="submit" class="btn btn-link nav-link buttonToLink">Log out</button>

```

Comment:
Could you not use the `border-0` class from Bootstrap instead of creating your own class? See [here](https://getbootstrap.com/docs/4.0/utilities/borders/)